lib/Doctrine/ORM/Mapping/Builder/ManyToManyAssociationBuilder.php

```diff
@@ -45,6 +45,18 @@ public function setJoinTable($name)
         return $this;
     }
 
+    /**
+     * @param bool $name
```

Comment:
That variable name looks wrong and anyway, you can use native type declarations here :)

Reply:
Right, the name is off 👍 For types I was trying to follow how other methods look like, but I can surely add type here if that'll be the consensus :)

Reply:
Yeah IMO we are going to migrate to types at some point, so I think we should make that as less work as possible.